KristianJBorgwarth/Golferino
# Lms/feat/create golfcourse

How to test?

Through swagger:
 * Create a location
 * Create a golfcourseid, by passing the locationid of the recently created location

Try and delete the location from you DB -- it should cascade leading to the golfcourse also getting deleted (??)
